dev/build_release.py

```diff
@@ -529,6 +511,14 @@ def init_argument_parser(cls, parser):
         '--bintray_repo', default='',
         help='Publish to this bintray repo.\n'
              'This requires BINTRAY_USER and BINTRAY_KEY are set.')
+      parser.add_argument(
```

Comment:
This `jar_repo` is the target repo for the microservice jars, not the dependency jars. For example, the jars produced from testing this script: https://bintray.com/spinnaker-team/jacobkiefer-jars. My understanding is all the dependency jars are downloaded to a local repository when the microservices are being built, and are pulled at whatever version is specified in `build.gradle`.